ckanext/chevrontheme/templates/snippets/package_item.html

```diff
@@ -21,14 +21,14 @@
 {% block package_item %}
   <tr>
     <td>
-      {{ h.link_to(package.chevron_dataset or 'Unknown', h.url_for(controller='package', action='read', id=package.name)) }}
+      {{ h.link_to(package.species_id or 'Unknown', h.url_for(controller='package', action='read', id=package.name)) }}
     </td>
     <td>
-      {{ package.specimen_type }}
+      {{ package.species_name }}
```

Comment:
this will break the table view for the other data types (e.g. EDMS)

If you need to use something different for PadiL, use the if/else pattern used elsewhere (e.g. line 30)